synapse/storage/databases/main/events.py

```diff
@@ -355,9 +355,9 @@ def _persist_events_txn(
         txn: LoggingTransaction,
         *,
         events_and_contexts: List[Tuple[EventBase, EventContext]],
-        inhibit_local_membership_updates: bool = False,
-        state_delta_for_room: Optional[Dict[str, DeltaState]] = None,
-        new_forward_extremities: Optional[Dict[str, Set[str]]] = None,
+        inhibit_local_membership_updates: bool,
+        state_delta_for_room: Dict[str, DeltaState],
+        new_forward_extremities: Dict[str, Set[str]],
```

Comment:
These are always passed (and are never `None`). `_persist_events_txn` only has one caller, not sure why this had defaults, etc.